spec/03-plugins/13-cors/01-access_spec.lua

```diff
@@ -591,7 +602,9 @@ for _, strategy in helpers.each_strategy() do
           method  = "OPTIONS",
           headers = {
             ["Host"]                           = "cors5.com",
+            ["Origin"]                         = "origin5.com",
             ["Access-Control-Request-Headers"] = "origin,accepts",
+            ["Access-Control-Request-Method"] = "GET",
```

Comment:
Would you mind pointing us to the test that asserts the behavior fixed by this PR (i.e. that non-preflight `OPTIONS` requests get properly proxied after a preflight request)?